activesupport/lib/active_support/encrypted_file.rb

```diff
@@ -49,6 +49,13 @@ def key
       read_env_key || read_key_file || handle_missing_key
     end
 
+    # Reads the file and returns the decrypted content.
+    #
+    # Raises:
+    # - MissingContentError if unable to decrypt content.
+    # - MissingKeyError if key is missing and <tt>raise_if_missing_key</tt> is true.
+    # - InvalidKeyLengthError if key length is invalid.
```

Comment:
`read` does not raise `InvalidKeyLengthError`.  (`write` / `encrypt` [does](https://github.com/rails/rails/blob/81c5c9971abe7a42a53ddbfede2683081a67e9d1/activesupport/lib/active_support/encrypted_file.rb#L87).)

```suggestion
```